### PR TITLE
Completed es6-const-let exercise. - Daniel Cho

### PR DESCRIPTION
**Pull request by Daniel-J-Cho**

### CODE REVIEW

**Comment by Daniel-J-Cho** [comment]
Screen Shot 2022-05-02 at 12 19 42 PM

File: es6-const-let/index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>es6-const-let</title>
+</head>
+<body>
+  <script>
+    var greetings = [
```
Comment (scbowler) [inline es6-const-let/index.html]: Missed one

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, except you missed one `var`. Let me know if you have any questions.

File: es6-const-let/index.html (hunk 1 above)
Comment (Daniel-J-Cho) [reply es6-const-let/index.html]: Screen Shot 2022-05-02 at 12 57 18 PM

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**